src/PoolManager.sol

```diff
@@ -279,21 +278,25 @@ contract PoolManager is IPoolManager, ProtocolFees, NoDelegateCall, ERC6909Claim
     }
 
     /// @inheritdoc IPoolManager
-    function settle(Currency currency) external payable override onlyWhenUnlocked returns (uint256 paid) {
+    function settle() external payable onlyWhenUnlocked returns (uint256 paid) {
+        Currency currency = CurrencyReserves.getSyncedCurrency();
+        // If not previously synced, expects native currency to be settled because currency.isNative() == address(0)
```

Comment:
```suggestion
        // If not previously synced, expects native currency to be settled because CurrencyLibrary.NATIVE == address(0)
```

Reply:
nit... but `currency.isNative()` returns a boolean, so will never be equal to address(0)

Reply:
good catch!